liana-gui/src/app/state/settings/bitcoind.rs

```diff
@@ -361,6 +361,7 @@ impl BitcoindSettings {
                     });
                 }
             }
+            view::SettingsEditMessage::Clipboard(text) => return clipboard::write(text),
```

Comment:
This will be needed also for `ElectrumSettings`.

Reply:
done